dotnet/aspnet/mssqldb/ArmTemplates/windows-webapp-sql-template.json

```diff
@@ -127,9 +127,7 @@
                     "location": "[parameters('databaseLocation')]",
                     "properties": {
                         "edition": "[parameters('databaseEdition')]",
-                        "collation": "SQL_Latin1_General_CP1_CI_AS",
-                        "maxSizeBytes": "2147483648",
-                        "requestedServiceObjectiveId": "dd6d99bb-f193-4ec1-86f2-43d3bccbc49c"
+                        "collation": "SQL_Latin1_General_CP1_CI_AS"
```

Comment:
are other properties not required?

Reply:
The removed one's are not required, since we are passing databaseEdition explicitly